src/mame/skeleton/css11501sk9.cpp

```diff
@@ -0,0 +1,136 @@
+PCMCIA slots for storage and one Gigabit Ethernet port (GBIC).
+
+Hardware components:
+
+Main board.
+ -Sipex SP3243ECA Intelligent RS-232 Transceiver.
+ -Crystal CrystalLAN CS8900A Ethernet ISA LAN Controller.
+ -Exar ST16C1551 UART.
+ -Microchip PIC16LF877PT next to a 10 MHz cristal (U35).
+ -Xilinx Spartan XC2S50 (U23).
+ -Atmel 24C02N SEEPROM (U11) next to the Xilinx Spartan XC2S50 at U23.
+ -Two PCMCIA slots for Microdrive or Flash Drive.
+ -Intel SPD6722QCCE PC-Card/PCMCIA controller.
+ -Two 24C04R6 SEEPROMs near the two PCMCIA slots (U25 and U32).
+ -Sipex 3223ECY RS-232 transceiver.
+ -ST M4T32-BR12SH6 Timekeeper.
+ -Xilinx Spartan XC2S50 (another one, U33).
+ -Another 24C02N (U28), near the Xilinx Spartan XC2S50 at U33.
+ -Vitesse VSC2102-08UQ Network Processor / Intelligent Packet Processor, near a 25 MHz crystal.
+ -Intel LXT9785EHC 8-port Fast Ethernet PHY Transceiver near a 50 MHz crystal.
+ -PMC RM7000A 400T 64-Bit MIPS RISC Microprocessor with Integrated L2 cache.
+ -AMD AM29LV3200B Flash Memory.
+ -PMC PM2329-BC.
+ -Vitesse VSC2708-00UR.
+ -AMCC S2068TB.
+ -Cypress CY7C1360B-166AC 9-Mbit (256K x 36) Pipelined SRAM.
+ -Samsung K7N403601B-QC13 128Kx36 Pipelined NtRAM.
+ -SORIMM slot with a 256MB RAM module (Samsung MS18R1628EH0-CM8CI).
+ -Cypress CY7C1360B-166AC 9-Mbit (256K x 36) Pipelines SRAM (another one).
+ -Intel SPD6722QCCE PC-Card/PCMCIA controller.
+ -Xilinx XC18V02 next to the first Xilinx Spartan XC2S50 (U23).
+
+Sub-board (SSL acceleration).
+ -Broadcom BCM5821A1KTB Super E-Commerce Processor (high-performance public-key processor) next to a 100 MHz crystal.
+ -Martel GT-64120A-BN-3 (System Controller for 7000 CPU) next to another 100 MHz crystal.
+ -PMC RM7000A 400T 64-Bit MIPS RISC Microprocessor with Integrated L2 cache.
+ -AMD AM29LV1600B Flash Memory.
+ -Altera Max EPM3128ATC100-5
+ -Broadcom BCM5700C2KPB PCI-X 10/100/1000 Base-T controller, near a 125 MHz crystal.
+ -Vitesse VSC2102-08UQ Network Processor / Intelligent Packet Processor, near a 25 MHz crystal.
+ -IDT 71V546S133PF 128K x 36, 3.3V Synchronous SRAM with ZBT Feature, Burst Counter and Pipelined Outputs.
+ -24C04R SEEPROM between the BCM5700C2KPB, the VSC2102-08UQ, and the 71V546S133PF.
+ -Microchip PIC16LF872 next to a 10 MHz crystal.
+ -Xilinx Spartan XC2S50.
+ -Xilinx 18V01JC In-System Programmable Configuration PROM next to the Xilinx Spartan XC2S50.
+ -SORIMM slot with a 64MB RAM module (Samsung MS18R1622EH0-CM8).
+ -SODIMM slot with a 512MB RAM module (Cisco CIS00-21160-111CD).
+
+**********************************************************************************************************************/
+
+#include "emu.h"
+#include "cpu/mips/mips3.h"
+
+
+namespace {
+
+class css11501sk9_state : public driver_device
+{
+public:
+	css11501sk9_state(const machine_config &mconfig, device_type type, const char *tag)
+		: driver_device(mconfig, type, tag)
+		, m_maincpu(*this, "maincpu")
+		, m_subcpu(*this, "subcpu")
```

Comment:
Given this is coming from a SSL board this may as well be named `sslcpu` instead.